sdk-commons/src/main/java/org/openmrs/maven/plugins/model/Server.java

```diff
@@ -313,15 +318,17 @@ private void replaceDbNameInDbUri() {
             setParam(Server.PROPERTY_DB_URI, dbUri);
         }
     }
-    public void addWatchedProject(Project project) {
-        linkProject(project);
-
+    public void addWatchedProject(Project project) throws MojoExecutionException {
         Set<Project> watchedProjects = getWatchedProjects();
+        Optional<Project> existingSameModule = watchedProjects.stream().filter(existingProject -> existingProject.getArtifactId().equals(project.getArtifactId())).findFirst();
+        if(existingSameModule.isPresent()) {
```

Comment:
https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-CodeStyle

Reply:
@dkayiwa could you elaborate more on this? I do not see what should be changed here.

Reply:
space after `if`

Reply:
Thank you. I fixed it.